src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlQueueClient.cs

```diff
@@ -152,41 +152,61 @@ public async Task<IReadOnlyCollection<JobInfo>> DequeueJobsAsync(byte queueType,
 
         public async Task<JobInfo> DequeueAsync(byte queueType, string worker, int heartbeatTimeoutSec, CancellationToken cancellationToken, long? jobId = null)
         {
-            try
+            const int MaxNumberOfIcMs = 3;
```

Comment:
😆 IcMs?

Reply:
HAHAHA I was probably speaking about IcMs with someone. Fixing it.